exotica/src/Tools.cpp

```diff
@@ -37,6 +37,8 @@
 #include <iostream>
 #include <typeinfo> //!< The RTTI Functionality of C++
 #include <cxxabi.h> //!< The demangler for gcc... this makes this system dependent!
+#include <regex>
```

Comment:
This will definitely make our gcc requirement >4.9, which we already require anyways to avoid segfaults from the wrapper. Want to flag it up either way (non-14.04 standard compiler).